assets/js/gutenberg-plugin.js

```diff
@@ -179,11 +179,23 @@ const DistributorPlugin = () => {
 		return null;
 	}
 
+	const distributorTopMenu = document.querySelector(
+		'#wp-admin-bar-distributor'
+	);
+
+	// eslint-disable-next-line no-shadow, react-hooks/rules-of-hooks -- permission checks are needed.
+	const post = useSelect( ( select ) =>
+		select( 'core/editor' ).getCurrentPost()
+	);
+	// Make the post title available to the top menu.
+	dtGutenberg.postTitle = post.title;
```

Comment:
```suggestion
	dt.postTitle = post.title;
```

Reply:
This is strange. If I modify the `const { document, dtGutenberg, MouseEvent } = window;` line and change it to `const { document, dtGutenberg, MouseEvent, dt } = window;`, `dt` is undefined. However if I don't make that modification I'm able to reference it in code directly via `dt.postTitle` or `window.dt.postTitle`. Going with `window.dt.postTitle` for now as it's more explicit and doesn't require an eslint exception, but please let me know if you have a better suggestion.